chips/sam4l/src/pm.rs

```diff
@@ -232,6 +235,15 @@ pub enum PBBClock {
 }
 
 #[derive(Copy,Clone)]
+pub enum PBCClock {
```

Comment:
I like deriving `Debug` for these kind of base types by default; saves annoying work later